test/unit/specs/store/governance/deposits.spec.js

```diff
@@ -34,16 +34,16 @@ describe(`Module: Deposits`, () => {
     })
     let { actions, state } = module
     let commit = jest.fn()
-    proposals.forEach(async (proposal, i) => {
-      let proposalId = proposal.proposal_id
+    const proposalIds = Object.keys(proposals)
+    proposalIds.forEach(async (proposal_id, i) => {
```

Comment:
you can just use `Object.forEach(function (value, key) {})` and delete the `proposalIds` const

```suggestion
    proposals.forEach((async (proposal, proposal_id) => {
```

Reply:
Not sure it's feasible to loop through object keys directly, instead I removed, where I wasn't using the array later, some of proposalIds declarations.